avocado/utils/configure_network.py

```diff
@@ -114,20 +114,20 @@ def add_ipaddr(self, ipaddr, netmask):
     def bring_down(self):
         """Utility used to Bring down interface """
 
-        cmd = "ifdown {}".format(self.name)
+        cmd = "ip link set {} down".format(self.name)
```

Comment:
They are very different. But this is related to a bigger change here. Before, the network information would be saved to the network-scripts file, which would be the source of truth to bring the interfaces up. Now you're setting all the information at run time and the network-scripts file is not even part of the flow, unless user manually calls `save()`.
Not sure what would be the best approach here, but if the network-scripts file is not used anymore as source of truth, it should be used at least to persist the current runtime configuration.

Reply:
Yes, I'm aware. This is part of the big refactoring: try to keep consistency here. Some methods were changing in runtime only, while other one were doing persistency.

So the idea of this big change is: everything is in runtime, unless the user decides to call `.save()`. And yes, there is room for improvments.